tests/main/listing/task.yaml

```diff
@@ -7,11 +7,14 @@ prepare: |
 execute: |
 
     echo "List prints core snap version"
+    # most core versions should be like "16-2", so [0-9]{2}-[0-9.]+
+    # but edge will have a timestamp in there, "16.2+201701010932", so add an optional \+[0-9]+ to the end
+    # *current* edge also has .git. and a hash snippet, so add an optional .git.[0-9a-f]+ to the already optional timestamp
     if [ "$SPREAD_BACKEND" = "linode" -o "$SPREAD_BACKEND" == "qemu" ] && [ "$SPREAD_SYSTEM" = "ubuntu-core-16-64" ]; then
         echo "With customized images the ubuntu-core snap is sideloaded"
-        expected="^core .* [0-9]{2}-[0-9.]+  +x[0-9]+ +- *$"
+        expected='^core .* [0-9]{2}-[0-9.]+(\+[0-9]+(\.git\.[0-9a-f]+)?)? +[0-9]+ +- *$'
```

Comment:
I think a 'x' is missing here for the revision (on the ubuntu-core system core is sideoaded https://github.com/snapcore/snapd/blob/master/tests/lib/prepare.sh#L265)